src/history/hs.c

```diff
@@ -364,7 +365,7 @@ __hs_insert_updates_verbose(WT_SESSION_IMPL *session, WT_BTREE *btree)
 static int
 __hs_insert_record_with_btree_int(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_BTREE *btree,
   const WT_ITEM *key, const WT_UPDATE *upd, const uint8_t type, const WT_ITEM *hs_value,
-  WT_HS_TIME_PAIR stop_ts_pair)
```

Comment:
You didn't add this here, but it's unusual to pass a structure as a parameter - we usuall pass as a pointer. Could you update this usage to do that? i.e: go from `WT_HS_TIME_POINT stop_time_point` to `WT_HS_TIME_POINT *stop_time_point` and similarly for below.